docs/concepts/llm_convenience_wrappers.md

```diff
@@ -118,6 +130,40 @@ chunk.delta    # original.choices[0].delta
 chunk.content  # original.choices[0].delta.content
 ```
 
+### Extraction
+
+Often you want to extract structured information into a format like JSON. The [`extract`](../api/chat/models.md#mirascope.chat.models.OpenAIChat.extract) method makes this extremely easy by extracting the information into a Pydantic `BaseModel` schema that you define:
+
+```python
+from mirascope import OpenAIChat
+from pydantic import BaseModel
+
+class BookInfo(BaseModel):
+    """Information about a book."""
+
+    title: str
+    author: str
+
+book_info = chat.extract("The Name of the Wind is by Patrick Rothfuss.", BookInfo)
+assert isinstance(book_info, BookInfo)
+assert book_info.model_dump() == {
+    "title": "The Name of the Wind",
+    "author": "Patrick Rothfuss",
+}
+```
+
+#### Retries
+
+Often you will want to retry your query in the event of a `ValidationError` when the model fails to properly populate the schema you've provided. Just set the number of retries and `extract` will automatically retry up to that many times (by default `retries` is `0`):
```

Comment:
when the model fails to properly populate the schema you've provided

to

when extract fails to convert the string response into a pydantic model

Reply:
Just (https://www.linkedin.com/pulse/stop-using-word-just-david-niebach-crm-mba-cpa-cga)

to

You can simply

Reply:
Hilarious that the author uses "just" in the first few sentences haha

Reply:
Done.